ehitchhiking-ui/src/app/shared/services/api.services/profile-modal.api.service.ts

```diff
@@ -0,0 +1,24 @@
+import {Injectable} from '@angular/core';
+import {ApiService} from './api.service';
+import {URL_REGISTRY} from '@shared/constants/urlRegistry';
+
+@Injectable()
+export class ProfileModalApiService {
+	constructor(private apiService: ApiService) {}
+
+	sendCarsListRequest(userId: string): Promise<any> {
```

Comment:
pls define types for params and return values

Reply:
done